### PR TITLE
Connected change-password

### PR DESCRIPTION
**Pull request by ElijahRW**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
     * Connected Client's password change function (in my-account component) to the service and API

### CODE REVIEW

**Review by Liriaene — APPROVED** [review]
Changes look goo to me.

File: src/app/my-account/my-account.component.ts [hunk 1]
```diff
@@ -56,25 +53,18 @@ export class MyAccountComponent implements OnInit {
...
-      this.pword = this.passFields.newPassField;
-      this.messages.message = 'Password Changed';
-      warn = false;
-      this.passColors.oldPass = this.colors.green;
+      this.apiUsersCrudService.changePassword(this.passFields.oldPassField, this.passFields.newPassField).subscribe(
+        (_) => {
+          this.messages.message = 'Password Changed';
+          warn = false;
+          this.passColors.oldPass = this.colors.green;
+        });
```
Comment (OkapalDominic) [inline src/app/my-account/my-account.component.ts:65]: If the old password doesn't match there should be a message and password field color change, similar to how validating the rest of it works.
```suggestion
        }, () => {
          this.passColors.oldPass = this.colors.red;
          this.messages.message = 'Old Password Is Invalid!';
        });
```

**Review by OkapalDominic — CHANGES_REQUESTED** [review]
The snackbar notification could be easily missed, and the component is already setup to easily give more visual feedback when the password is wrong.

**Comment by ElijahRW** [comment]
Removing PR for updated branch that fixes double commits

**Pull request closed without merging**